readme.txt

```diff
@@ -205,6 +205,7 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 * Fix - Resolved an issue where Order Status was not populated when exporting the Attendee List using Tickets Commerce. [ET-1883]
 * Tweak - Update default footer text of Tickets Emails to include link to website. [ET-1971]
 * Fix - Update usage of `method_exists()` to comply with PHP 8.1 standards. [ET-1759]
+* Fix - Update button will now show when the opt-out checkbox shows on the My Tickets page. [ET-1980]
```

Comment:
does the new filter `tribe_tickets_plus_hide_attendees_list_optout` need to appear also in the readme?

Reply:
Great, thanks for clarification @codingmusician !